portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,33 @@
+package main.java.com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        // Get the value entered in the form.
+        String textValue = request.getParameter("text-input");
+        String textValue2 = request.getParameter("text-input2");
+        String textValue3 = request.getParameter("text-input3");
+
+        // Print the value so you can see it in the server logs.
+        System.out.println("You submitted: " + textValue);
+        System.out.println("You submitted: " + textValue2);
+        System.out.println("You submitted: " + textValue3);
+
+        // Write the value to the response so the user can see it.
+        response.getWriter().println("You submitted: " + textValue);
+        response.getWriter().println("You submitted: " + textValue2);
+        response.getWriter().println("You submitted: " + textValue3);
```

Comment:
optional 🤷 : Here it's not really necessary, but if you find yourself writing something many times, consider capturing it in a variable, then reusing that variable like.

```
String response = "You submitted: \n Name: " + request.getParameter("name") + .... "\n\n\nRedirecting" ;
// Print the values so you can see it in the server logs.        
System.out.println(response);
// Write the value to the response so the user can see it.
response.getWriter().println(response);
//....
```


You'll especially see how useful this is as code evolves. For example, suppose you want to change the format or add more data. Instead of having to do it twice - System.out and  getWriter().println - you just change the `response` String and you're done.